torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2701,6 +2702,10 @@ def create_unbacked_symint(self):
 
         return SymInt(SymNode(symbol, self, int, None, fx_node=fx_node))
 
+    @record_shapeenv_event()
```

Comment:
I don't think you need this cc @ysiraichi

Reply:
Yep. No need for recording here, since it doesn't modify `ShapeEnv` state.